IndyAV/simulation/indyav_gazebo/xacro/wheels.xacro

```diff
@@ -2,93 +2,98 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro" name="wheels">
   <xacro:include filename="$(find mil_gazebo)/xacro/util.xacro"/>
 
-  <xacro:property name="wheel_length" value="0.25"/>
-  <xacro:property name="wheel_width" value="0.25"/>
-  <xacro:property name="wheel_height" value="0.25"/>
+  <xacro:property name="wheel_radius" value="0.125"/>
+  <xacro:property name="wheel_width" value="0.125"/>
   <xacro:property name="wheel_mass" value="5"/>
 
-  <xacro:macro name="wheel_geometry">
+  <xacro:macro name="wheel_geometry">	  
     <geometry>
-      <box size="${wheel_length} ${wheel_width} ${wheel_height}"/>
+       <cylinder length="${wheel_width}" radius="${wheel_radius}"/>
     </geometry>
   </xacro:macro>
 
-  <xacro:macro name="wheel_surface">
-    <surface>
-      <friction>
-        <ode>
-          <mu>1000</mu> <!-- left / right  friction ,very high -->
-          <mu2>1</mu2> <!-- forwards / backwards friction, very low -->
-          <fdir1> 0 1 0 </fdir1>
-        </ode>
-      </friction>
-    </surface>
-  </xacro:macro>
-
   <xacro:macro name="do_wheel_surface" params="name">
     <gazebo reference="${name}_link">
-      <collision>
-        <xacro:wheel_surface/>
-      </collision>
+      <mu1>100</mu1>
```

Comment:
a comment about what these are / do and where they cam from would be appreciated